be/src/storage/tablet_updates.cpp

```
@@ -2806,6 +2806,21 @@ size_t TabletUpdates::_get_rowset_num_deletes(const Rowset& rowset) {
     return num_dels;
 }
 
+void TabletUpdates::_get_extra_file_size(int64_t* pindex_size, int64_t* col_size) {
+    for (const auto& entry : fs::directory_iterator(_tablet.schema_hash_path())) {
```

From [docs](https://en.cppreference.com/w/cpp/filesystem/directory_iterator/directory_iterator), this function may throw an exception. But you ignore the exception, which will cause the process to crash.
So please handle the exception and return status rather than the void.

---

Done